packages/json-extension/src/index.tsx

```diff
@@ -23,6 +23,12 @@ const CSS_CLASS = 'jp-RenderedJSON';
  * The MIME type for JSON.
  */
 export const MIME_TYPE = 'application/json';
+// NOTE: not standardized yet
```

Comment:
If we delay merging this PR until 4.1, is there a chance these would be standardized by then, or is it not worth waiting for it?

Reply:
not worth waiting, i doubt this will be standardized soon

Reply:
Hi, I am working on helping to standardize `application/jsonl` MIME_TYPE, but I have no experience in doing so, so I confirm it will take me a bit of time to write the RFC and submit it, so it's not worth waiting.